cmd/juju/application/bundle.go

```diff
@@ -601,6 +607,16 @@ func (h *bundleHandler) addApplication(change *bundlechanges.AddApplicationChang
 		return errors.Annotatef(err, "cannot deploy application %q", p.Application)
 	}
 	h.writeAddedResources(resNames2IDs)
+
+	if h.bundleURL != nil {
+		tag := names.NewApplicationTag(p.Application).String()
+		result, err := h.api.SetAnnotation(map[string]map[string]string{tag: {"bundleURL": h.bundleURL.String()}})
+		if err == nil && len(result) > 0 {
+			err = result[0].Error
+			logger.Debugf("error setting bundleURL annotation for %q: %s", p.Application, err)
```

Comment:
You need to check the error before logging it.

Reply:
done, and removed the additional layer of indentation.